Scripts/travis.sh

```diff
@@ -61,12 +66,14 @@ execute_xcodebuild() {
 }
 
 if [ "${TYPE}" == "RUBY" ]; then
-  rvm use 2.2.2;
+  rvm use 2.2.5;
   cd gem;
   bundle install --retry=3;
   rake;
 elif [ "${TYPE}" == "UNIT" ]; then
   execute_xcodebuild Tests/UnitTests/UnitTests.xcodeproj EarlGreyUnitTests
+elif [ "${TYPE}" == "FUNCTIONAL" && "${FILTER}" != "" ]; then
```

Comment:
Perhaps a better way would be:
Make the 3rd parameter of execute_xcodebuild be the filter (i.e. ${3}). Then in `execute_xcodebuild` method, check if ${3} is set, create a new variable assigned to`-only-testing ${3}` and pass it to `xcodebuild test` command.